LightweightCharts/Classes/LightweightChartsModels/ChartColor.swift

```diff
@@ -1,3 +1,76 @@
 import Foundation
+import UIKit
 
-public typealias ChartColor = String
+public enum ChartColor {
+    case color(UIColor)
+    case string(String)
+}
+
+// MARK: - ExpressibleByStringLiteral
+extension ChartColor: ExpressibleByStringLiteral {
+
+    public init(stringLiteral value: String) {
+        self = .string(value)
+    }
+
+}
+
+// MARK: - _ExpressibleByColorLiteral
+extension ChartColor: _ExpressibleByColorLiteral {
+
+    public init(_colorLiteralRed red: Float, green: Float, blue: Float, alpha: Float) {
+        let color = UIColor(
+            red: CGFloat(red),
+            green: CGFloat(green),
+            blue: CGFloat(blue),
+            alpha: CGFloat(alpha)
+        )
+        self = .color(color)
+    }
+
+}
+
+// MARK: - Decodable
+extension ChartColor: Decodable {
+
+    public init(from decoder: Decoder) throws {
+        let stringValue = try decoder.singleValueContainer().decode(String.self)
+        self = .string(stringValue)
```

Comment:
Can we decode to `UIColor`? Every deserialization will replace all `UIColors` with string value `"rgba(x,x,x,x)"`